### ISSUE TITLE
[question] How is reliable packet delay handled?

### ISSUE THREAD

**Issue by SynergiAios** [RevenantX/LiteNetLib#93]
I will setup a test scenario for this next weekend, so I thought, I could ask it in before hand to check how you designed your framework for this. So the question is targeting the following:

How are reliable packages handled, that the receiving client isn't ACKing for whatever reason?

Is there a built in queue that will build up for every client till a certain threshold and then start dropping old packages?

Will the server completly stop sending packages till every client has ACKed the package?

Basically how is this handled in your framework?

tia for taking the time and maybe answering my questions.

**Comment by RevenantX** [comment]
>Is there a built in queue that will build up for every client till a certain threshold and then start dropping old packages?
I used technique like in TCP. There is special window for that. No packet drops.
`public const int DefaultWindowSize = 64;`
64 packets in special queue that waiting for acks. Other queued packets in another queue that has no limits.
If client doesn't send any packet for some time (bigger than DisconnectTimeout) - it will be disconnected.
